pkg/plugins/golang/v3/init.go

```diff
@@ -35,7 +35,7 @@ import (
 
 // Variables and function to check Go version requirements.
 var (
-	goVerMin = golang.MustParse("go1.17.9")
+	goVerMin = golang.MustParse("go1.18")
```

Comment:
@oscr I think the problem here is that we need to put the value with 0. So, it needs to be 1.18.0